src/wallet/rpcdump.cpp

```diff
@@ -783,6 +783,11 @@ UniValue z_importviewingkey(const UniValue& params, bool fHelp)
             "2. rescan             (string, optional, default=\"whenkeyisnew\") Rescan the wallet for transactions - can be \"yes\", \"no\" or \"whenkeyisnew\"\n"
             "3. startHeight        (numeric, optional, default=0) Block height to start rescan from\n"
             "\nNote: This call can take minutes to complete if rescan is true.\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"type\" : \"xxxx\",                         (string) \"sprout\" or \"sapling\"\n"
+            "  \"address\" : \"address|DefaultAddress\",    (string) The address(sprout) or the DefaultAddress(sapling)\n"
```

Comment:
This wording is unclear.

```suggestion
            "  \"address\" : \"address\",                   (string) The address corresponding to the viewing key (for Sapling, this is the default address).\n"
```

Reply:
Changed at https://github.com/zcash/zcash/pull/4426/commits/24a75fe4f88e948d0cf9ac643f9b8b788f2a18eb